vllm/utils.py

```diff
@@ -608,6 +598,16 @@ def make_tensor_with_pad(
     return torch.tensor(padded_x, dtype=dtype, device=device)
 
 
+def str_to_int_tuple(s: str) -> Tuple[int, ...]:
```

Comment:
nit: it seems that the only thing that changed here is the position of the function in the file. If this could be moved it would make the diff cleaner

Reply:
Thanks, fixed in latest push.